src/modules/api/user/link/link.controller.ts

```diff
@@ -53,4 +59,23 @@ export class LinkController {
       return { status: 'NOT OK', error: error.message };
     }
   }
+  @Put(':id')
+  async editLink(
+    @Param('userId') userId: string,
```

Comment:
`userId` is not used anywhere.

Reply:
To test it in the swagger i had to use it

Reply:
okay, but can we remove it now?

Reply:
i guess we can

Reply:
Okay, let's remove it.